app/views/TaxCreditsView.scala.html

```diff
@@ -32,9 +32,8 @@ <h1 @if(!isIvrRedirect){class="header-margin-top"}>@messages("tax.credits.title"
 
     <div class="govuk-inset-text">
         <p>@messages("global.opening.times.title")</p>
-        <p>@messages("global.opening.times.mon.sat.8to10")</p>
-        <p>@messages("global.opening.times.sun.9to10")</p>
-        <p>@messages("global.closed.time.bankholidays")</p>
+        <p>@messages("global.opening.times.mon.sat.8to8")</p>
+        <p>@messages("global.closed.time.sun.banckholidays")</p>
```

Comment:
should be `bankholidays`

Reply:
Ticket states 

Opening times:
Monday to Saturday, 8am to 8pm
Closed Sundays and bank holidays.

So this is correct

Reply:
sorry mate you are correct I meant to change the spelling of banckholidays @tonymyers